### PR TITLE
Create turabian-fullnote-bibliography-no-ibid

### PR DESCRIPTION
**Pull request by dhacker29**
Removes the ibid function from Turabian style as it is not preferred by many institutions and it also has the issue of using ibid for the first reference on a page which should not happen.

### CODE REVIEW

**Comment by bwiernik** [comment]
Any particular reason not to just use Chicago (full note)?

**Comment by dhacker29** [comment]
Professor requires us to use Turabian style.

**Comment by bwiernik** [comment]
Turabian style is just slightly simplified Chicago.

**Comment by bwiernik** [comment]
@adam3smith what do you think?

**Comment by adam3smith** [comment]
I think the experience of Turabian is dumb, but since that's inevitable, the style makes sense

File: turabian-fullnote-bibliography-no-ibid.csl [hunk 1]
```diff
@@ -0,0 +1,811 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Turabian 8th edition (full note) no ibid</title>
```
Comment (bwiernik) [inline turabian-fullnote-bibliography-no-ibid.csl:4]:
```suggestion
    <title>Turabian 8th edition (full note, no ibid)</title>
```

**Comment by bwiernik** [comment]
Thanks!

**Pull request merged**